src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/__tests__/ChartAxisConfiguration.test.tsx

```diff
@@ -0,0 +1,521 @@
+      target: { value: '2014_AY' },
+    });
+
+    await userEvent.type(
+      referenceLinesSection.getByLabelText('Label'),
+      'I am label',
+    );
+
+    userEvent.click(screen.getByRole('button', { name: 'Add line' }));
+
+    await waitFor(() => {
+      const rows = referenceLinesSection.getAllByRole('row');
+      expect(rows).toHaveLength(3);
+      expect(rows[1]).toHaveTextContent('2014_AY');
+      expect(rows[1]).toHaveTextContent('I am label');
+      expect(within(rows[1]).getByRole('button', { name: 'Remove' }));
+    });
+
+    userEvent.click(screen.getByRole('button', { name: 'Save chart options' }));
+
+    await waitFor(() => {
+      const formValues: AxisConfiguration = {
+        dataSets: [],
+        groupBy: 'timePeriod',
+        min: 0,
+        max: undefined,
+        referenceLines: [
+          {
+            label: 'I am label',
+            position: '2014_AY',
+          },
+        ],
+        showGrid: true,
+        size: 50,
+        sortAsc: true,
+        sortBy: 'name',
+        tickConfig: 'default',
+        tickSpacing: 1,
+        type: 'major',
+        visible: true,
+        unit: '',
+        label: {
+          text: '',
+        },
+      };
+      expect(handleSubmit).toHaveBeenCalledWith(formValues);
+    });
+  });
+
+  test('removing reference lines', async () => {
+    const handleSubmit = jest.fn();
+
+    render(
+      <ChartBuilderFormsContextProvider initialForms={testFormState}>
+        <ChartAxisConfiguration
+          id="chartBuilder-major"
+          type="major"
+          configuration={testAxisConfigurationWithReferenceLines}
+          definition={verticalBarBlockDefinition}
+          data={testTable.results}
+          meta={testTable.subjectMeta}
+          onChange={noop}
+          onSubmit={handleSubmit}
+        />
+      </ChartBuilderFormsContextProvider>,
+    );
+
+    const referenceLinesSection = within(
+      screen.getByRole('table', { name: 'Reference lines' }),
+    );
+    expect(referenceLinesSection.getAllByRole('row')).toHaveLength(3);
+
+    userEvent.click(
+      referenceLinesSection.getByRole('button', { name: 'Remove' }),
+    );
+
+    await waitFor(() => {
+      const rows = referenceLinesSection.getAllByRole('row');
+      expect(rows).toHaveLength(2);
+    });
+
+    userEvent.click(screen.getByRole('button', { name: 'Save chart options' }));
+
+    await waitFor(() => {
+      const formValues: AxisConfiguration = {
+        dataSets: [],
+        groupBy: 'timePeriod',
+        min: 0,
+        max: undefined,
+        referenceLines: [],
+        showGrid: true,
+        size: 50,
+        sortAsc: true,
+        sortBy: 'name',
+        tickConfig: 'default',
+        tickSpacing: 1,
+        type: 'major',
+        visible: true,
+        unit: '',
+        label: {
+          text: '',
+        },
+      };
+      expect(handleSubmit).toHaveBeenCalledWith(formValues);
```

Comment:
Similarly, would probably suggest testing the submission without a reference line separately